src/Modules/Factory/ModuleFactoryImpl1.cc

```diff
@@ -186,13 +185,11 @@ void ModuleDescriptionLookup::addEssentialModules()
   addModuleDesc<BuildMappingMatrix>("Real ported module", "...");
   //addModuleDesc<ImportDatatypesFromMatlab>("Improved version of Matlab importer", "work in progress"); //not ready yet
   addModuleDesc<FlipSurfaceNormals>("FlipSurfaceNormals","ChangeMesh","SCIRun","...","...");
-  addModuleDesc<BuildNoiseColumnMatrix>("BuildNoiseColumnMatrix","Math","SCIRun","...","...");
-  addModuleDesc<ComputeSVD>("ComputeSVD","Math","SCIRun","...","...");
-  
+  addModuleDesc<BuildNoiseColumnMatrix>("BuildNoiseColumnMatrix", "Math", "SCIRun", "...", "...");
   addModuleDesc<EditMeshBoundingBox>("Rewrite", "...");
   addModuleDesc<ConvertIndicesToFieldData>("Real ported module", "..."); 
   addModuleDesc<SolveInverseProblemWithTikhonov>("...", "...");
-  addModuleDesc<ShowColorMapModule>("ShowColorMap", "Visualization", "SCIRun", "Real ported module", "..."); 
```

Comment:
@mhansen1 Bad merge here--you removed two modules. Please fix. There's no nice way to verify merging this file now, other than to avoid it. You can add the addModuleDesc line to ModuleFactoryImpl2.cc instead, less traffic there. The one way to test is to watch the module count in the GUI, it should go up by one only.